doc/static/js/ncs.js

```diff
@@ -101,15 +118,21 @@ function NCS () {
   state.showVersion = function() {
     const ncs = window.NCS;
     const VERSIONS = ncs.versions.VERSIONS;
+    const last_version_index = ncs.last_version_index;
     const path_suffix = "/" +  ncs.current_page;
-    const last_release_url = ncs.url_root + VERSIONS[1] + path_suffix;
+    const last_release_url = ncs.url_root + VERSIONS[last_version_index] + path_suffix;
     const latest_release_url = ncs.url_root + "latest" + path_suffix;
 
+    const SWITCH_MSG = "You might want to switch to the documentation for " +
+      "the <a href='" + last_release_url + "'>" + VERSIONS[last_version_index] +
+      "</a> release or the <a href='" + latest_release_url + "'>current " +
+      "state of development</a>."
+
     const OLD_RELEASE_MSG =
-      "You are looking at an older version of the documentation. You might " +
-      "want to switch to the documentation for the <a href='" +
-      last_release_url + "'>" + VERSIONS[1] + "</a> release or the " +
-      "<a href='" + latest_release_url + "'>current state of development</a>.";
+      "You are looking at an older version of the documentation. " + SWITCH_MSG;
+
+    const DEV_RELEASE_MSG =
+      "You are looking at a development version of the documentation. " + SWITCH_MSG;
 
     const LAST_RELEASE_MSG =
       "You are looking at the documentation for the last official release.";
```

Comment:
```suggestion
      "You are looking at the documentation for the latest official release.";
```
Unrelated to this PR, but we got feedback that "last" is misleading.

Reply:
Could you change it here, or should I do a separate PR? ;)